Library/Homebrew/test/support/fixtures/internal_tap_json/homebrew-core/Formula/f/fennel.rb

```diff
@@ -0,0 +1,25 @@
+class Fennel < Formula
```

Comment:
Might be nicer to strip these down to only the bits that are actually used in the API (e.g. less in `def install`, `test do`)

Reply:
Good point.

Reply:
Apparently removing them causes the tests to fail for some reason. 🤷

Since it's not that important, I'll just leave them as is.

Reply:
Sorry, I don't mean remove those blocks, poor communication on my part, but instead make the contents of each e.g. a one-liner/comment.

Reply:
Yeah, that's what I tried and the tests started failing at that point. Not obvious to me what difference it makes though and the diff output seemed pretty much identical.

Reply:
Ok, no worries, ignore it!